docs/tempo/website/configuration/tls.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Configure TLS Communication
+weight: 70
+---
+
+# Configure TLS Communication
+
+Tempo can be configured to communicate between the components using Transport Layer Security, or TLS.
+
+Please note, the ciphers and TLS version here are for example purposes only. We are in no way recommending which ciphers or TLS versions for use in production environments, and are here for example purposes only.
+
+## Server Configuration
+
```

Comment:
I'm adding an introductory sentence here to summarize what the YAML snippet does and/or how it would be used. What is the purpose of this code block? 
Please feel free to update this phrasing. Where would people add this tempo config file or some place else? 
```suggestion
## Server configuration

This sample TLS server configuration shows supported options. 

```